Riot/AppDelegate.m

```diff
@@ -3278,6 +3279,16 @@ - (void)addMatrixCallObserver
                                                                                          NSLog(@"[AppDelegate] presentCallViewController");
                                                                                          [self presentCallViewController:NO completion:nil];
                                                                                      }
+                                                                                     else if (call.state == MXCallStateEnded)
+                                                                                     {
+                                                                                         // Set call vc to nil to let our app handle new incoming calls even it wasn't killed by the system
+                                                                                         
+                                                                                         if (weakSelf)
+                                                                                         {
+                                                                                             typeof(self) self = weakSelf;
```

Comment:
MXStrongifyAndReturnIfNil can be used instead of this. You can see examples in code.